controller/auth-controller.js

```diff
@@ -0,0 +1,63 @@
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+const User = require("../models/user");
+
+exports.login = async (req, res) => {
+  const { name, pass } = req.body;
+  const key = process.env.JWT_SECRET;
+
+  const tempUser = await User.findOne({ userName: name });
```

Comment:
You can use `user` instead of `tempUser`, as you've used `User` for requiring user model.

Reply:
done